Makefile

```diff
@@ -20,7 +20,7 @@ export GOBIN
 
 # Docker
 IMAGEDIR=$(BASE)/images
-DOCKERFILE=$(CURDIR)/Dockerfile
+DOCKERFILE?=$(CURDIR)/Dockerfile
```

Comment:
so this change is to allow `DOCKERFILE` be configured manually(e.g. to Dockerfile.ppc64le) when running `make`?

Reply:
yes, yes, another idea would be `make image_ppc64le`, not sure which one would be better

Reply:
I'm fine with either way, perhaps worth document the build steps later in developer doc.